### PR TITLE
resolves #15 allow punctuation marks

### LINKED ISSUE #15: The positive look ahead should include punctuation marks (, ! ? : ;)
Currently, the regular expression use only space, dot followed by a space, dot at the end of the line and end of line:

https://github.com/sapegin/textlint-rule-stop-words/blob/3d1d56bcb8bd5d157ca2269fe8a63b6cb944a0a4/index.js#L124

In other words, it won't match in the following cases:

```
add up?
add up!
add up, and subtract.
add up; and subtract.
add up: 10
```

Using word boundary (`\b`) is probably a bad idea but adding common punctuation marks seems reasonable. What do you think?

### PR DESCRIPTION
**Pull request by Mogztter**
resolves #15

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -121,7 +121,14 @@ function filterDict(rules, excludedWords) {
  */
 function getRegExp(word) {
 	const wordPattern = escapeRegExp(word).replace(/'/g, "['’‘]");
-	return new RegExp(`(?:^|[^-\\w])(${wordPattern}(?= |\\. |\\.$|$))`, 'ig');
+	const punctuations = ['\\.', ',', ':', ';', '\\?', '!'];
+	const punctuationsRx = punctuations.map(
```
Comment (sapegin) [inline index.js:125]: `Rx` is quite obscure: 

```suggestion
	const punctuationsRegExp = punctuations.map(
```

**Review by sapegin — COMMENTED** [review]
Thanks! I've left one minor comment.

File: index.js (hunk 1 above)
Comment (Mogztter) [reply index.js]: Done :+1:

**Pull request merged**